charts/datadog/README.md

```diff
@@ -782,7 +782,14 @@ helm install <RELEASE_NAME> \
 | datadog.securityAgent.runtime.fimEnabled | bool | `false` | Set to true to enable Cloud Workload Security (CWS) File Integrity Monitoring |
 | datadog.securityAgent.runtime.network.enabled | bool | `true` | Set to true to enable the collection of CWS network events |
 | datadog.securityAgent.runtime.policies.configMap | string | `nil` | Contains CWS policies that will be used |
-| datadog.securityAgent.runtime.securityProfile.enabled | bool | `false` | Set to true to enable CWS runtime anomaly detection |
+| datadog.securityAgent.runtime.securityProfile.driftEvents.defaultMinimumStablePeriod | string | `"6h"` | Set the minimum amount of time to set a security-profile as stable |
+| datadog.securityAgent.runtime.securityProfile.driftEvents.enabled | bool | `false` | Set to true to enable CWS runtime drift events |
+| datadog.securityAgent.runtime.securityProfile.driftEvents.minimumStablePeriod.dns | string | `"25h"` | Set the minimum amount of time to set a dns events as stable |
+| datadog.securityAgent.runtime.securityProfile.driftEvents.minimumStablePeriod.exec | string | `"6h"` | Set the minimum amount of time to set an exec events as stable |
+| datadog.securityAgent.runtime.securityProfile.driftEvents.rateLimiter.numEventsAllowed | int | `20` | Set the number of drift events events allowed per duration |
+| datadog.securityAgent.runtime.securityProfile.driftEvents.rateLimiter.numKeys | int | `1000` | Set the number of keys in the rate limiter |
+| datadog.securityAgent.runtime.securityProfile.driftEvents.rateLimiter.period | string | `"5m"` | Set the rate limiter period of drift events |
```

Comment:
I'm not sure we want to expose "internal" parameters to customers here?
Also, I found naming them under "driftEvents" here and "anomaly_detection" later on could be confusing.